components/brave_new_tab_ui/components/default/braveToday/customize/ChannelCard.tsx

```diff
@@ -20,30 +26,27 @@ const Container = styled(Flex) <{ backgroundColor: string }>`
   color: white;
   position: relative;
 
-  :hover {
-    opacity: 0.8;
+  &[data-channel-card-is-followed=true] {
+    &:not(:hover, :focus-within) ${SubscribeButton} {
+      opacity: 0;
+    }
   }
 `
 
-const SubscribeButton = styled(FollowButton)`
-    position: absolute;
-    top: 8px;
-    right: 8px;
-`
-
 interface Props {
-  channelId: string
+  channelName: string
```

Comment:
Good call, in my original iteration I thought channels might have a bit more info - nice to keep things simple.

Reply:
Yeah we still need to do translation and images and I thought I was going to do that in this PR, but this still makes sense I suppose for now.